.github/workflows/integration-tests-smoke.yml

```
@@ -79,24 +70,24 @@ jobs:
           echo "CUSTOM_CORE_REF=${CL_REF}" >> "${GITHUB_ENV}"
         env:
           CL_REF: ${{ github.event.inputs.cl_branch_ref }}
-      - name: Build Image ${{ matrix.image.name }}
+      - name: Build Image
         if: steps.check-image.outputs.exists == 'false'
-        uses: smartcontractkit/.github/actions/ctf-build-image@c4705bfdbf6c8e57c080d82a3c4f013aa96a2dfb # v2.3.19
+        uses: smartcontractkit/.github/actions/ctf-build-image@7b28f2c598c26a8a0af8818b64d5344247752c71 # v0.1.3
```

Can use the tag directly now:

```suggestion
        uses: smartcontractkit/.github/actions/ctf-build-image@ctf-build-image/0.1.3
```